content/efcore/getting-started/what-is-ef-core.md

```diff
@@ -0,0 +1,5 @@
+# What is Entity Framework Core?
```

Comment:
@rdahlman3 and I had a discussion about this, and we think that "Intro to EF Core" better-matches the other chapter titles in this tutorial and the C# tutorial. For example, the C# tutorial has an "Intro to LINQ" chapter.